src/OrchardCore/OrchardCore.FileStorage.AzureBlob/BlobStorageOptions.cs

```diff
@@ -13,8 +13,16 @@ public abstract class BlobStorageOptions
         public string ContainerName { get; set; }
 
         /// <summary>
-        /// The base directory path to use inside the container for this stores contents.
+        /// The base directory path to use inside the container for this store's content.
         /// </summary>
-        public string BasePath { get; set; }
+        public string BasePath { get; set; } = "";
+
+        /// <summary>
+        /// Returns a value indicating whether the basic state of the configuration is valid.
+        /// </summary>
+        public virtual bool IsValid()
```

Comment:
`IsConfigured()` instead. This means that the configuration exists, but it mean that the provides values are invalid.